TACC/Core-Styles
# fix: right-align a lone login form action

## Overview

Right-align a login-style form action that is all alone.

## Related

- supports https://github.com/tapis-project/authenticator/pull/42

## Changes

- **adds** style for solitary login form actions

## Testing

1. Open http://localhost:3000/components/detail/s-form-page--login.
2. Verify "Create Account" and "Log In" button layout is unchanged.
3. Delete the "Create Account" link under the fields.
4. Verify "Log In" button is aligned to the right.

## UI

| 3.1 | 3.2. | 2. & 4. |
| - | - | - |
| ![1 delete link](https://github.com/TACC/Core-Styles/assets/62723358/a4f34561-1eae-4581-81a2-75be239b10c5) | ![2 only button](https://github.com/TACC/Core-Styles/assets/62723358/f061a040-b369-4662-810d-568df524f72a) | ![3 one and two both align well](https://github.com/TACC/Core-Styles/assets/62723358/045dc129-a2cd-4f1a-b894-0be4f10a059c) |
